erpnext/manufacturing/doctype/job_card/job_card.js

```diff
@@ -235,4 +257,14 @@ frappe.ui.form.on('Job Card Time Log', {
 		frm.set_value('job_started', 0);
 		frm.set_value('started_time', '');
 	}
-})
\ No newline at end of file
+});
+
+frappe.ui.form.on('BOM Scrap Item', {
+	stock_qty: (frm, cdt, cdn) => {
+		frm.events.set_scrap_amount(frm, cdt, cdn);
```

Comment:
This is throwing an error on the BOM document

2 options here:

- Either add a dummy `set_scrap_amount` function in `bom.js`, or
- Check if a `set_scrap_amount` function exists and only then call it